main.js

```diff
@@ -0,0 +1,282 @@
+      console.log(dataMovies);
+      if (dataMovies.Response === 'False') {
+        return error();
+      }
+      document.querySelector(".wrap-title").innerHTML = "";
+      errorResponse.innerHTML = '';
+      dataMovies.Search.forEach(renderHTMLpage);
+      pagination();
+    });
+    
+}
+
+const searchBtn = document.querySelector(".search-btn");
+searchBtn.addEventListener("click", mySearch);
+
+function renderHTMLpage(data) {
+  const wrapItem = document.createElement("div");
+  const titleImg = document.createElement("img");
+  const titleInfo = document.createElement("div");
+  const titleName = document.createElement("h3");
+  const titleYear = document.createElement("p");
+  const titleType = document.createElement("p");
+  let btnMore = document.createElement("button");
+
+  titleName.innerHTML = `${data.Title}`;
+  titleYear.innerHTML = `Release year: ${data.Year}`;
+  titleType.innerHTML = `Type: ${data.Type}`;
+  btnMore.innerHTML = `more`;
+  titleImg.src = data.Poster;
+
+  wrapTitle.append(wrapItem);
+  wrapItem.append(titleImg, titleInfo);
+  titleInfo.append(titleName, titleYear, titleType, btnMore);
+
+  wrapItem.classList.add("box-item");
+  titleInfo.classList.add("about-movi");
+  titleName.classList.add("header-movi");
+  titleYear.classList.add("text-movi");
+  titleType.classList.add("text-movi");
+
+  btnMore.classList.add("btn-more");
+  btnMore.setAttribute("id", data.imdbID);
+  const imdbID = data.imdbID;
+
+  btnMore.addEventListener("click", moreInfo);
+
+  function moreInfo(e) {
+    const omdbapiMovieURL = `http://www.omdbapi.com/?apikey=${IPA_key}&i=${imdbID}`;
+    console.log(e.currentTarget.value);
+    fetch(omdbapiMovieURL)
+      .then((result) => result.json())
+      .then((data) => {
+        renderHTMLmoreInfo(data);
+        btnMore.setAttribute("disabled", true);
+      });
+  }
+
+  function renderHTMLmoreInfo(data) {
+    const runTime = document.createElement("p");
+    const actors = document.createElement("p");
+    const boxOffice = document.createElement("p");
+    const country = document.createElement("p");
+    const director = document.createElement("p");
+    const genre = document.createElement("p");
+    const language = document.createElement("p");
+    const plot = document.createElement("p");
+
+    runTime.innerHTML = `${data.Runtime}`;
+    actors.innerHTML = `${data.Actors}`;
+    boxOffice.innerHTML = `${data.BoxOffice}`;
+    country.innerHTML = `${data.Country}`;
+    director.innerHTML = `${data.Director}`;
+    genre.innerHTML = `${data.Genre}`;
+    language.innerHTML = `${data.Language}`;
+    plot.innerHTML = `${data.Plot}`;
+
+    runTime.classList.add("text-movi");
+    actors.classList.add("text-movi");
+    boxOffice.classList.add("text-movi");
+    country.classList.add("text-movi");
+    director.classList.add("text-movi");
+    genre.classList.add("text-movi");
+    language.classList.add("text-movi");
+    plot.classList.add("text-movi");

```

Comment:
`.classList.add("text-movi")` repeats a lot. It can be optimized

For example

`const htmlData = ['runTime', 'actors', 'boxOffice'...].forEach(... add class in callback)`